### PR TITLE
update raft log gc related tikv configurations

### PR DESCRIPTION
**Pull request by tabokie**
Signed-off-by: tabokie <xy.tao@outlook.com>

### What is changed, added or deleted? (Required)

Refer to https://github.com/tikv/tikv/pull/10599

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -472,11 +472,17 @@ Configuration items related to Raftstore
 + Minimum value: `0`
 + Unit: MB|GB
 
+### `raft-log-compact-sync-interval`
```
Comment (en-jin19) [inline tikv-configuration-file.md:475]:
```suggestion
### `raft-log-compact-sync-interval` <span class="version-mark">New in v5.3</span>
```

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -496,6 +502,12 @@ Configuration items related to Raftstore
 + Default value: 3/4 of the Region size
 + Minimum value: greater than `0`
 
+### `raft-log-reserve-max-ticks`
```
Comment (en-jin19) [inline tikv-configuration-file.md:505]:
```suggestion
### `raft-log-reserve-max-ticks` <span class="version-mark">New in v5.3</span>
```

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -496,6 +502,12 @@ Configuration items related to Raftstore
 + Default value: 3/4 of the Region size
 + Minimum value: greater than `0`
 
+### `raft-log-reserve-max-ticks` <span class="version-mark">New in v5.3</span>
+
++ Old Raft logs could be reserved if `raft-log-gc-threshold` is not reached. GC them after `raft-log-reserve-max-ticks` ticks.
```
Comment (en-jin19) [inline tikv-configuration-file.md:507]:
```suggestion
+ If the maximum allowable count of residual Raft logs did not reach the value of `raft-log-gc-threshold`, old Raft logs are reserved. In this case, after `raft-log-reserve-max-ticks` is set, TiKV performs garbage collection (GC) to those Raft logs.
```

Comment (tabokie) [reply tikv-configuration-file.md:507]: Could you improve based on this new version?
"Performs garbage collection (GC) to residual Raft logs after `raft-log-reserve-max-ticks` ticks, even if their count doesn't reach the value of `raft-log-gc-threshold`."
And the "ticks" refers to elapsed time.

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -472,11 +472,17 @@ Configuration items related to Raftstore
 + Minimum value: `0`
 + Unit: MB|GB
 
+### `raft-log-compact-sync-interval` <span class="version-mark">New in v5.3</span>
+
++ The time interval to compact unnecessary Raft logs.
```
Comment (en-jin19) [inline tikv-configuration-file.md:477]:
```suggestion
+ The time interval to compact unnecessary Raft logs
```

File: tikv-configuration-file.md [hunk 5]
```diff
@@ -496,6 +502,12 @@ Configuration items related to Raftstore
 + Default value: 3/4 of the Region size
 + Minimum value: greater than `0`
 
+### `raft-log-reserve-max-ticks` <span class="version-mark">New in v5.3</span>
+
++ After the number of ticks set by this configuration item passed, even if the number of residual Raft logs does not reach the value set by `raft-log-gc-threshold`, TiKV still performs garbage collection (GC) to them.
```
Comment (TomShawn) [inline tikv-configuration-file.md:507]:
```suggestion
+ After the number of ticks set by this configuration item passes, even if the number of residual Raft logs does not reach the value set by `raft-log-gc-threshold`, TiKV still performs garbage collection (GC) to these logs.
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Review by hicqu — APPROVED** [review]

File: tikv-configuration-file.md [hunk 6]
```diff
@@ -496,6 +502,12 @@ Configuration items related to Raftstore
 + Default value: 3/4 of the Region size
 + Minimum value: greater than `0`
 
+### `raft-log-reserve-max-ticks` <span class="version-mark">New in v5.3</span>
+
++ After the number of ticks set by this configuration item passes, even if the number of residual Raft logs does not reach the value set by `raft-log-gc-threshold`, TiKV still performs garbage collection (GC) to these logs.
++ Default value: `6`
++ Minimum value: greater than `0`
```
Comment (TomShawn) [inline tikv-configuration-file.md:509]:
```suggestion
+ Minimum value: greater than `0` 
```

**Pull request merged**